src/gui/Gui.cc

```diff
@@ -152,12 +159,6 @@ std::unique_ptr<ignition::gui::Application> createGui(
   // Configuration file from command line
   if (_guiConfig != nullptr && std::strlen(_guiConfig) > 0)
   {
-    if (!app->LoadConfig(_guiConfig))
-    {
-      ignwarn << "Failed to load config file[" << _guiConfig << "]."
-              << std::endl;
-    }
```

Comment:
I think this block can't be removed. Passing a custom `--gui-config` isn't working right now, either during simulation or playback.

Reply:
Thanks for catching this, I don't believe I meant to remove it, I added it back 13dd0e626c31a43529d2078830e8601d998094bd